client/state/selectors/test/is-site-using-full-site-editing.js

```diff
@@ -0,0 +1,57 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import isSiteUsingFullSiteEditing from '../is-site-using-full-site-editing';
+
+describe( 'isSiteUsingFullSiteEditing', () => {
```

Comment:
Noice! ✨ Thanks for adding tests

Reply:
Thanks for the nudge - I get lazy sometimes 🤪

Reply:
I usually only ask for state related pieces, as units are a great fit for catching bugs/testing interfaces.

Component tests/e2e is more of a judgement call, for usefulness + critical path vs overhead.